share/functions/fish_config.fish

```diff
@@ -19,6 +19,14 @@ function fish_config --description "Launch fish's web based configuration"
         set -lx __fish_bin_dir $__fish_bin_dir
         if set -l python (__fish_anypython)
             $python "$__fish_data_dir/tools/web_config/webconfig.py" $argv
+
+            # If the execution of 'webconfig.py' fails, display python location and return.
+            if test $status -ne 0
+                echo "Please check if Python has been installed successfully."
+                echo "You can find the location of Python by executing the 'command -s python $python' command."
```

Comment:
`$python` is still there.

Reply:
It should be `command -s $python`, bacause $python maybe python/python3

Reply:
No, $python won't be defined outside of this.

If you want you can do `command -s python python3`.

Reply:
Are you sure we can't access $python? This python variable is defined above. I tried it, and it worked. However, I'm not very familiar with Fish. If you're certain, I'll modify it to `command -s python python3`.
![image](https://github.com/fish-shell/fish-shell/assets/19884146/96c87cf5-86af-4143-a430-18be18b7d281)

Reply:
It's a local variable - `set -l`.

That means once the function returns it'll be gone.

If the user runs `command -s $python` outside, it won't be the same $python.

Reply:
I used a wrong python3.exe, and got this output.
![image](https://github.com/fish-shell/fish-shell/assets/19884146/bfe388eb-6759-4ae0-aac4-02220bdb6b56)

The echo output is: 
```
fish_config
Please check if Python has been installed successfully.
You can find the location of Python by executing the 'command -s python3' command.
```
`$python` will be replaced by the python command used by the user to execute the script. So users won't use $python 
 in the command line, did I misunderstand?